cmd/eksctl/main.go

```diff
@@ -31,6 +35,17 @@ func addCommands(rootCmd *cobra.Command, flagGrouping *cmdutils.FlagGrouping) {
 }
 
 func main() {
+
+	// Log internally-logged kubernetes errors uniformly
+	logK8sError := func(err error) {
```

Comment:
We don't use `k8s` in variable names, use `kube` if you must have something short.